python/lsst/cp/pipe/cpFunctions.py

```diff
@@ -91,15 +91,6 @@ def findInputs(self, datasetTypeName, inputTypeString=None):
                                                self.inputCollection)
         self.instrument = self.inputValues[0].dataId['instrument']
 
-        try:
-            existingValues = list(self.registry.queryDimensions(['calibration_label'],
-                                                                instrument=self.instrument,
-                                                                calibration_label=self.calibrationLabel))
-            if len(existingValues) != 0:
-                raise RuntimeError(f"Existing entries for calibration_label {self.calibrationLabel}")
-        except LookupError:
-            pass
-
```

Comment:
Why is this code now disappearing? If it's not supposed to be there, don't add it in the first place.

Reply:
Peculiarities with SQLAlchemy is the short answer.  I thought the query would raise LookupError when called, but it only raises when the result is used.  The repack into an array does this.

Reply:
Sorry, that comment was more about a lack of orthogonality in the commits than the code.